apps/crn-frontend/src/network/users/UserList.tsx

```diff
@@ -34,6 +35,12 @@ const UserList: React.FC<UserListProps> = ({
     searchQuery,
     filters: new Set(),
   });
+  usePrefetchWorkingGroups({
+    currentPage: 0,
+    pageSize,
+    searchQuery,
+    filters: new Set(),
```

Comment:
I guess it shouldn't apply the filters for users since it's prefetching the list to show it faster on a different tab (with diff filters). Groups and Teams are doing the same.

Reply:
shouldn't all usePrefetch have filter as `new Set()` then? In the TeamList I saw that I was the one who changed from `filters: new Set()` to `filters` 🙈 

originally they were set to `filters: new Set()` https://github.com/yldio/asap-hub/pull/795/files although we keep the search term when we change tabs. Like If I type `test` in the search bar when `Teams` is selected, if I click in the `Interest Groups` tab then it still filters by `test`. So I'm not sure which one it should be, but for me it would make sense for them all to be the same 🤔

Reply:
I see, it seems UserList is the only one that uses `new Set()`. I think it's fine to change it then